Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -1290,37 +1358,37 @@ variable (R)
 
 The weights are applied using `•`; typically this definition is used either with `S = R` or
 `[Algebra S R]`, although this is stated more generally. -/
-def weightedSumSquares [Monoid S] [DistribMulAction S R] [SMulCommClass S R R] (w : ι → S) :
-    QuadraticForm R (ι → R) :=
-  ∑ i : ι, w i • proj i i
-#align quadratic_form.weighted_sum_squares QuadraticForm.weightedSumSquares
+def weightedSumSquares [Monoid S] [DistribMulAction S R] [SMulCommClass S R R] [SMulCommClass R S R]
```

Comment:
This extra `SMulCommClass` is a bit suspicious

Reply:
Looks like you've fixed this for me: https://github.com/leanprover-community/mathlib4/pull/7569/commits/3c60a70c7d69cf01cd7d2d6494335d4bc5a93e68 - thanks.